### DISCUSSION TITLE
CocoaPods could not find compatible versions for pod "GoogleMLKit/BarcodeScanning"

### DISCUSSION THREAD

**Discussion by MattWellsBlue42** [capawesome-team/capacitor-mlkit#208] [Q&A] [answered]
I have been trying to update @capacitor-firebase/authentication and such to make it compatible with the new Apple privacy manifest stuff and I have been following some of the other discussions about this topic and I have gotten to running npm with the next version

```
npm install @capacitor-mlkit/barcode-scanning@next
npx cap sync
```

It downloads 7.0.0-next.1 but when I run npx cap sync I get the following error:
```
[capacitor] [error] Analyzing dependencies
[capacitor]         [!] CocoaPods could not find compatible versions for pod "GoogleMLKit/BarcodeScanning":
[capacitor]         In Podfile:
[capacitor]         CapacitorMlkitBarcodeScanning (from `../../node_modules/@capacitor-mlkit/barcode-scanning`) was resolved to 7.0.0-next.1, which depends on
[capacitor]         GoogleMLKit/BarcodeScanning (= 7.0.0)
[capacitor]         
[capacitor]         Specs satisfying the `GoogleMLKit/BarcodeScanning (= 7.0.0)` dependency were found, but they required a higher minimum deployment target.
```

And I can't quite figure out what to do to fix it.

Thank you!

**Comment by robingenz** [comment, marked as answer]
Please take a look at the [Breaking Changes](https://github.com/capawesome-team/capacitor-mlkit/blob/next/packages/barcode-scanning/BREAKING.md#minimum-deployment-target). You need to increase the minimum deployment target.

**Reply by MattWellsBlue42** [reply]
Perfect! That worked. Thank you so much for the helpful and quick reply
